app/views/job_orders/invoice.html.erb

```diff
@@ -0,0 +1,224 @@
+<!doctype html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <title>Job Order Invoice</title>
+
+  <style>
+      .invoice-box {
+          max-width: 800px;
+          margin: auto;
+          padding: 30px;
+          border: 1px solid #eee;
+          box-shadow: 0 0 10px rgba(0, 0, 0, .15);
+          font-size: 16px;
+          line-height: 24px;
+          font-family: 'Helvetica Neue', 'Helvetica', Helvetica, Arial, sans-serif;
+          color: #555;
+      }
+
+      .invoice-box table {
+          width: 100%;
+          line-height: inherit;
+          text-align: left;
+      }
+
+      .invoice-box table td {
+          padding: 5px;
+          vertical-align: top;
+      }
+
+      .invoice-box table tr td:nth-child(2) {
+          text-align: right;
+      }
+
+      .invoice-box table tr.top table td {
+          padding-bottom: 20px;
+      }
+
+      .invoice-box table tr.top table td.title {
+          font-size: 45px;
+          line-height: 45px;
+          color: #333;
+      }
+
+      .invoice-box table tr.information table td {
+          padding-bottom: 40px;
+      }
+
+      .invoice-box table tr.heading td {
+          background: #eee;
+          border-bottom: 1px solid #ddd;
+          font-weight: bold;
+      }
+
+      .invoice-box table tr.details td {
+          padding-bottom: 20px;
+      }
+
+      .invoice-box table tr.item td {
+          border-bottom: 1px solid #eee;
+      }
+
+      .invoice-box table tr.item.last td {
+          border-bottom: none;
+      }
+
+      .invoice-box table tr.total td:nth-child(2) {
+          border-top: 2px solid #eee;
+          font-weight: bold;
+      }
+
+      @media only screen and (max-width: 600px) {
```

Comment:
same as before, should we use BS breakpoints to keep things consistent?

Reply:
In this case, it's not really relevant since this will be converted to a PDF, I just followed guidelines I found online concerning emails and PDFs, I'll leave them that way just so they look the same as before